cmd/acmesolver/go.mod

```diff
@@ -37,8 +37,8 @@ require (
 	go.uber.org/multierr v1.11.0 // indirect
 	go.uber.org/zap v1.25.0 // indirect
 	golang.org/x/net v0.17.0 // indirect
-	golang.org/x/sys v0.14.0 // indirect
-	golang.org/x/text v0.13.0 // indirect
+	golang.org/x/sys v0.15.0 // indirect
+	golang.org/x/text v0.14.0 // indirect
```

Comment:
These got upgraded as a side effect of my running `go get golang.org/x/crypto@v0.17.0` in *every* sub-module.
`go mod tidy` then removes the unused import and leaves behind these updates to indirect dependencies which *are* used in this sub-module.

I considered reverting these changes, but decided that it was neater to have all the sub-modules using the same versions of `golang.org/x/sys` and `golang.org/x/text`.